src/Tools/penaltyScoreCalc.js

```diff
@@ -0,0 +1,13 @@
+const penaltyScore = ({ result }) => {
```

Comment:
consider naming functions starting with a verb. Can't we pass the array in the argument so you don't need to destructure it?